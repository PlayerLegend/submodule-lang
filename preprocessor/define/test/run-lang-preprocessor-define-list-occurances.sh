#!/bin/sh

cat src/lang/preprocessor/define/test/lang-preprocessor-define-list-occurances.asdf.tree | $DEBUG_PROGRAM test/lang-preprocessor-define-list-occurances asdf
