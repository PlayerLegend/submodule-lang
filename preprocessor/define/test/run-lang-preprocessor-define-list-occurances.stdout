2 21
3 6
3 44
3 76
6 6
6 26
